website/content/plugins/drivers/podman.mdx

```diff
@@ -134,6 +134,15 @@ The `podman` driver implements the following [capabilities](/nomad/docs/concepts
 
 ## Task Configuration
 
+- `apparmor_profile` - (Optional) Name of a apparmor profile to be used instead of
+  the default profile. The special value unconfined disables apparmor for this container.
```

Comment:
```suggestion
- `apparmor_profile` - (Optional) Name of an AppArmor profile to use instead of
  the default profile. The special value `unconfined` disables AppArmor for this container.
```

Style: active voice (to be used -> to use)

and capitalized the product name